papaya-web/src/main/java/pl/edu/agh/papaya/mappers/SprintMapper.java

```diff
@@ -0,0 +1,48 @@
+package pl.edu.agh.papaya.mappers;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import pl.edu.agh.papaya.api.model.SprintDto;
+import pl.edu.agh.papaya.model.Sprint;
+
+@Component
+public class SprintMapper extends Mapper<Sprint, SprintDto> {
+
+    @Autowired
+    private LocalDateTimePeriodMapper localDateTimePeriodMapper;
+
+    @Autowired
+    private SprintStateMapper sprintStateMapper;
+
+    public List<SprintDto> mapToApi(List<Sprint> modelItems, LocalDateTime evaluationTime) {
+        return modelItems.stream()
+                .map(modelSprint -> mapToApi(modelSprint, evaluationTime))
+                .collect(Collectors.toList());
+    }
+
+    @Override
+    public List<SprintDto> mapToApi(List<Sprint> modelItems) {
```

Comment:
Dobra, lekka zmiana podejścia - ta klasa już oficjalnie nie ma żadnych profitów z tej abstrakcyjnej, ale trudno - jednak fajniej, jak cała lista jest ewaluowana względem tego samego momentu w czasie

Reply:
jak to nie ma profitów? :open_mouth: -- _polimorfizm_

Reply:
Znaczy no takie ideologiczne są, przepraszam, _polimorfizm_ trzeba szanować